HDFOutputer.cc

```diff
@@ -105,6 +124,16 @@ get_prods_and_sizes(std::vector<product_t> & input,
 void 
 HDFOutputer::output(EventIdentifier const& iEventID, 
                     std::vector<SerializerWrapper> const& iSerializers) {
+#ifdef H5_TIMING_ENABLE
+  size_t total_data_size = 0;
+#endif
+  char *p = getenv("HEP_IO_TYPE");
+  int method = hdf_method;
+
+  if ( p != NULL ) {
+    method = atoi(p);
```

Comment:
Doing this evaluation ever write will just slow things down. Much better to be done in the constructor based on the configuration information passed to the constructor.